assets/xml/archives/map_name_static.xml

```diff
@@ -1,20 +1,21 @@
 <Root>
     <File Name="map_name_static.unarchive">
-        <Blob Name="map_name_static_Blob_000000" Size="0x0400" Offset="0x0" />
-        <Blob Name="map_name_static_Blob_000400" Size="0x0400" Offset="0x400" />
-        <Blob Name="map_name_static_Blob_000800" Size="0x0400" Offset="0x800" />
-        <Blob Name="map_name_static_Blob_000C00" Size="0x0400" Offset="0xC00" />
-        <Blob Name="map_name_static_Blob_001000" Size="0x0400" Offset="0x1000" />
-        <Blob Name="map_name_static_Blob_001400" Size="0x0400" Offset="0x1400" />
-        <Blob Name="map_name_static_Blob_001800" Size="0x0400" Offset="0x1800" />
-        <Blob Name="map_name_static_Blob_001C00" Size="0x0400" Offset="0x1C00" />
-        <Blob Name="map_name_static_Blob_002000" Size="0x0400" Offset="0x2000" />
-        <Blob Name="map_name_static_Blob_002400" Size="0x0400" Offset="0x2400" />
-        <Blob Name="map_name_static_Blob_002800" Size="0x0400" Offset="0x2800" />
-        <Blob Name="map_name_static_Blob_002C00" Size="0x0400" Offset="0x2C00" />
-        <Blob Name="map_name_static_Blob_003000" Size="0x0400" Offset="0x3000" />
-        <Blob Name="map_name_static_Blob_003400" Size="0x0400" Offset="0x3400" />
-        <Blob Name="map_name_static_Blob_003800" Size="0x0400" Offset="0x3800" />
-        <Blob Name="map_name_static_Blob_003C00" Size="0x0400" Offset="0x3C00" />
+        <Texture Name="gGreatBayPointNameENGTex" OutName="great_bay_point_name_eng" Format="ia4" Width="128" Height="16" Offset="0x0" />                <!-- REGION_GREAT_BAY -->
```

Comment:
Perhaps make `PointName` a common prefix for them all?

Reply:
So as a Prefix instead of a Suffix? I suppose that would be more consistent with how we do textures in general, but that would also apply to the icon_items and the icons in parameter_static. I'll take a look at those too

Reply:
Ahh, I didn't notice. Looks like OoT `ItemIcon` for a common prefix for things like that. (see icon_item_static)

Reply:
I was thinking it would look nicer as a prefix instead of a suffix too